src/arcaflow_plugin_sdk/schema.py

```diff
@@ -5507,7 +5507,7 @@ def _validate_discriminator(self):
                 raise BadArgumentException(
                     f"the type of object id {set_member.id}'s discriminator"
                     f' field "{self.discriminator_field_name}" does not match'
-                    " OneOfSchema discriminator type; expected"
+                    " the discriminator type for the union; expected"
```

Comment:
Here we have "discriminator type" whereas in `test_schema.py` you changed the same "OneOfSchema discriminator" to "discriminated union type". Consistency would be nice.

Reply:
i think this is fixed